### ISSUE TITLE
Training days time

### ISSUE THREAD

**Issue by khawar512** [zhongyy/Face-Transformer#2]
I am doing training as same as your GPU Tesla V100. How many days are required for one epoch?

**Comment by zhongyy** [comment]
It seems that it takes about 5 days to train a model.

**Comment by zhongyy** [comment]
> I am doing training as same as your GPU Tesla V100. How many days are required for one epoch?

It seems that it takes about 5 days to train a model.

**Comment by khawar512** [comment]
Thank you @zhongyy

**Comment by khawar512** [comment]
Hi, 

I am doing it again and currently, the performance is below 60 in all datasets and batch count is

```
highest_acc: [0.5375, 0.5388333333333333, 0.5048333333333334, 0.517, 0.562142857142857, 0.5076666666666666]
Epoch 1 Batch 17650	Speed: 5.85 samples/s	Training Loss 33.7908 (33.8089)	Training Prec@1 0.000 (0.000)
Epoch 1 Batch 17660	Speed: 212.87 samples/s	Training Loss 33.6953 (33.7547)	Training Prec@1 0.000 (0.000)
Learning rate 0.000001
Perform Evaluation on ['lfw', 'talfw', 'calfw', 'cplfw', 'cfp_fp', 'agedb_30'] , and Save Checkpoints...
(12000, 512)
```


How many days does it take to train completely?

**Comment by zhongyy** [comment]
> Hi,
> 
> I am doing it again and currently, the performance is below 60 in all datasets and batch count is
> 
> ```
> highest_acc: [0.5375, 0.5388333333333333, 0.5048333333333334, 0.517, 0.562142857142857, 0.5076666666666666]
> Epoch 1 Batch 17650	Speed: 5.85 samples/s	Training Loss 33.7908 (33.8089)	Training Prec@1 0.000 (0.000)
> Epoch 1 Batch 17660	Speed: 212.87 samples/s	Training Loss 33.6953 (33.7547)	Training Prec@1 0.000 (0.000)
> Learning rate 0.000001
> Perform Evaluation on ['lfw', 'talfw', 'calfw', 'cplfw', 'cfp_fp', 'agedb_30'] , and Save Checkpoints...
> (12000, 512)
> ```
> 
> How many days does it take to train completely?

My training speed is about 280 samples/s. The acc on test set (LFW ...) will not be high since the lr of the first epoch is 0.000001 (warmup).

**Comment by XinWangg** [comment]
@zhongyy Can you share your training log?

**Comment by arunsubk** [comment]
Hi @zhongyy @khawar-islam  Strangely, thought my train loss after 5 epochs comes down to about 7, I still see highest_accuracy at 0.5 and model checkpoint is not saving any model after the first epoch (where too the highest accuracy was 0.5). Asking this because from the log shared above in this thread, @khawar-islam  was able to see highest_accuracy move to 0.53 etc on the first epoch itself. Could I be doing something wrong? I made no changes...just entered "lfw" as the input parameter? Please see the logs below ;this is scary because i am wondering if after all epoch my checkpoints wont get saved :(   

Learning rate 0.000299                                                                                         
Perform Evaluation on ['lfw'] , and Save Checkpoints...                                                        
(12000, 512)                                                                                                   
[lfw][198000]XNorm: 13.68387                                                                                   
[lfw][198000]Accuracy-Flip: 0.50000+-0.00000                                                                   
[lfw][198000]Best-Threshold: 0.00000                                                                           
highest_acc: [0.5]                                                                                             
Epoch 5 Batch 198010    Speed: 15.55 samples/s  Training Loss 6.1584 (6.5197)   Training Prec@1 38.281 (35.078)Learning rate 0.000299                                                                                         
Perform Evaluation on ['lfw'] , and Save Checkpoints...                                                        
(12000, 512)                                                                                                   
[lfw][198000]XNorm: 13.68387                                                                                   
[lfw][198000]Accuracy-Flip: 0.50000+-0.00000                                                                   
[lfw][198000]Best-Threshold: 0.00000                                                                           
highest_acc: [0.5]                                                                                             
Epoch 5 Batch 198010    Speed: 15.55 samples/s  Training Loss 6.1584 (6.5197)   Training Prec@1 38.281 (35.078)
